Makefile

```diff
@@ -31,6 +33,9 @@ build:
 		--file=Dockerfile \
 		$(CURDIR)/
 
+run: build
```

Comment:
What do you think about naming this target to `shell` instead of `run` to avoid any language misunderstanding (for non native speakers)?

Reply:
Fine by me!